### ISSUE TITLE
[Safari] SyntaxError: Invalid regular expression: invalid group specifier name

### ISSUE THREAD

**Issue by davidbailey00** [othree/smartypants.js#12]
Safari doesn't support lookbehind in regular expressions (see [Can I use](https://caniuse.com/js-regexp-lookbehind)) resulting in an error in 0.1.2 and newer:

```
SyntaxError: Invalid regular expression: invalid group specifier name
    RegExp
    EducateQuotes — smartypants.es6.js:437
    SmartyPants — smartypants.es6.js:172
```

The error occurs here: https://github.com/othree/smartypants.js/blob/master/smartypants.es6.js#L437

**Comment by othree** [comment]
Thanks for reporting. Should be fixed in 0.1.6.

**Comment by davidbailey00** [comment]
Thanks!
